### DISCUSSION TITLE
Group 7: DACA

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#163] [Group Projects Feedback]
Hi Group 7, please make sure to add a short title so we can reference to your project by name! This is a wonderful topic and I think it is very well motivated, so, as with Group 4, you should not spend too much time re-treading the literature on the importance of labor and immigration policy: there is plenty of work on this, so brief acknowledgement is sufficient. This is good for you, as it will leave you lots of room to approach the difficult problems of data and comparisons, which I will address here.

Data collection for this problem is extremely challenging. While it is true that you have a well-defined control group that is not eligible for DACA, collecting data on them is another matter. You should expect massive "natural" imbalances (ie, ineligible people leave the country) as well as response imbalances (since ineligible people aren't necessarily allowed to be in the US). So, for this kind of "natural experiment" design, where are you drawing your sample from in a manner that would be unbiased? Do you really expect that "administrative data" will have extensive and accurate records on employment _for people who cannot legally work_?

The second part of the challenge here is the data itself. What do you do with someone (eligible or ineligible) who no longer lives in the US, if they are employed elsewhere? What makes someone "employed" in your data when it's not legal for them to work (they probably won't be telling CPS about it)? Is that an artifact of data or of reporting? And, in what sense do you think any results would be "surprising", like, _of course_ people who are allowed to work will be working more, so what's the research value here? At least in that sense you might think about looking to other outcomes such as migration, family formation, etc, or looking up to aggregate outcomes at city/state/etc levels based on the number of people eligible for DACA in various locations.

To resolve problems like this, usually you need to give up some generality or representativeness in exchange for "identification" (a concept we will discuss a bit later in the class). What this means is coming up with a clever way to find eligible and ineligible people who won't have differentially biased outcomes by the mechanism of the treatment. For example, you might look at one or more cities that do not provide information on undocumented people to federal authorities against those that do. You might try to use databases of non-driving identification cards (which some cities and states issue regardless of status) to provide a functional pool of research subjects. You might also build your candidate pool by surveying for out-migration at, say, the point of origin, and although this would be complex, it can work!

You might also move away from individual-level designs and move to things like census-block or urban-area comparisons (particularly across state lines where immigration enforcement might vary dramatically _for parents of eligible individuals) to try and see if things change in areas when (presumably) larger numbers of their population got work authorization through DACA. However of course there are other differences in labor policy across cities and states that might have originally attracted those groups of people, so those comparisons need to be really carefully developed.

In sum -- this is a great challenge for you and I encourage you to carry it through. Don't focus now on covariates and controls -- and forget about timeline. Research like this takes years. For now, take the mindset -- "if I had a lab, what would be the best random experiment I could run" and focus on thinking of the best approximation of that you might be able to find in the real world, since you can't experiment on people after all. You have a great start with the DOB cutoff -- but now you have to add your own innovation of how to find the right people and measure interesting outcomes! Looking forward to seeing where this goes.

**Comment by BeaLeydier** [comment]
Hi team

Love this topic and your design! As Ben said, it's great to see some projects ask more macro questions that are widely studied by looking into them through the lense of a specific policy decision. It's a place where your ToC and market failure is already well identified, and all the challenge and fun will be in your research design and measurement !

A few things to build off Ben's comments 

1. You may need to think of other eligibility criteria for DACA to be able to mirror them on the comparison group you build; for example, criteria around the age they entered the US as, the education criteria, or the no criminal record criteria  
 
2. One challenge if you stick to identifying individuals from your sample into CPS is how you will identify the undocumented immigrants in general, and the DACA eligible ones in particular. While CPS does include immigrants (included undocumented ones) at least in theory, it doesn't include any info about the residency status of an individual, and it also doesn't include any identifiable information about the individual. You could identify immigrants using country of birth variables, but I don't think (unless I'm missing some other relevant context) that it'd be easy to distinguish between documented and undocumented, and even more so between DACA or not. So I agree with Ben that more aggregated analysis may be needed, and more specific or targeted data sources or locations may be what helps you.

3. To expand your research question from "do people who have legal status work more than people who don't" as Ben suggests, you could also be looking at compositional effects - to what extent the types of jobs change and how does or doesn't affect the labor market equilibirum in a local, more or less closed economy system for example.

Very cool topic and looking forward to how you fine tune your design on it!

Cheers,

**Comment by bbdaniels** [comment]
# Concept Note Review

Hi team! Thanks for the work on revising this -- however, it still feels a little incomplete, and I feel this is because of the challenges in the data that you have identified in your note. My main concern is that the model as written has just 100 observations (10 states over 10 years), and that is just not going to work! You have the raw materials here, and this is a very challenging project. Therefore to get it to a workable state, you will need to get a bit more creative with how you develop comparison groups, which I will provide some concrete suggestions on below. I particularly like the fact that you have taken the treatment variable and used a continuous version (DACA participation rate). Unfortunately you have not defined this in your note so I do not quite know how to interpret that, so you should go into a bit more detail since that's your main treatment.

Let's brainstorm ways to have better comparisons. I do not think that "imputing zeros" is a very useful approach. There is no variation there so everything is going to drop out. How about an approach that takes a sector-state-year approach? This way you can do things like compare across sectors (things like capital-intensive and labor-intensive sectors, which you already show an understanding of). This will allow you to go "wide" in the time series data, as well as to use all 50 states from 2013 onwards. This is a much more effective study if it is well designed!

So, my suggestion is to go back and revise that, and be very clear about the number of observations you might be able to obtain (sectors) using the data that you can get access to. Show how you might be interested in differences-in-differences between different sectors in the same state, and how you might be interested in across-state differences in the same sector. Finally, make sure you can account for clustering of treatment -- I expect that you will have regional co-variation across states, for example, so make sure you define the treatment variable very precisely and understand how it might work. You will be able to simulate precisely these elements using the tools from the next lecture, and I look forward to the final version incorporating these elements! Keep it up!
